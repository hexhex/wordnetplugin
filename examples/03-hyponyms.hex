%% this returns all hyponyms of a word
%% in all parts of speech

%% the search number for hyponyms is 3

word("work").
hyponyms(W,SW,H,SH) :- &wordnet[3,W](1,SW,H,SH),word(W).

hyponyms("work",1,"coursework",27), 
hyponyms("work",1,"task",26), 
hyponyms("work",1,"project",26), 
hyponyms("work",1,"mission",23), 
hyponyms("work",2,"masterpiece",7), 
hyponyms("work",3,"sailing",4), 
hyponyms("work",7,"writing",1)?
