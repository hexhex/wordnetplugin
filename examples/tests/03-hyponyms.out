{hyponyms("work",1,"coursework",27), hyponyms("work",1,"task",26), hyponyms("work",1,"project",26), hyponyms("work",1,"undertaking",26), hyponyms("work",1,"timework",25), hyponyms("work",1,"spadework",24), hyponyms("work",1,"missionary_work",23), hyponyms("work",1,"mission",23), hyponyms("work",1,"duty",22), hyponyms("work",1,"tending",21), hyponyms("work",1,"aid",21), hyponyms("work",1,"attention",21), hyponyms("work",1,"care",21), hyponyms("work",1,"investigating",20), hyponyms("work",1,"investigation",20), hyponyms("work",1,"substituting",19), hyponyms("work",1,"subbing",19), hyponyms("work",1,"toil",18), hyponyms("work",1,"labour",18), hyponyms("work",1,"labor",18), hyponyms("work",1,"labor",26), hyponyms("work",1,"social_service",17), hyponyms("work",1,"welfare_work",17), hyponyms("work",1,"paperwork",16), hyponyms("work",1,"nightwork",15), hyponyms("work",1,"unfinished_business",14), hyponyms("work",1,"loose_end",14), hyponyms("work",1,"logging",13), hyponyms("work",1,"make-work",12), hyponyms("work",1,"busywork",12), hyponyms("work",1,"ironing",11), hyponyms("work",1,"housekeeping",10), hyponyms("work",1,"housework",10), hyponyms("work",1,"housewifery",9), hyponyms("work",1,"heavy_lifting",8), hyponyms("work",1,"polishing",7), hyponyms("work",1,"shining",7), hyponyms("work",1,"service",6), hyponyms("work",1,"procedure",5), hyponyms("work",1,"operation",5), hyponyms("work",1,"job",3), hyponyms("work",1,"job",4), hyponyms("work",1,"action",2), hyponyms("work",1,"lavation",1), hyponyms("work",1,"washing",1), hyponyms("work",1,"wash",1), hyponyms("work",2,"publication",16), hyponyms("work",2,"workpiece",15), hyponyms("work",2,"work_in_progress",14), hyponyms("work",2,"woodwork",13), hyponyms("work",2,"caning",12), hyponyms("work",2,"wickerwork",12), hyponyms("work",2,"wicker",12), hyponyms("work",2,"silverwork",11), hyponyms("work",2,"polychrome",10), hyponyms("work",2,"openwork",9), hyponyms("work",2,"metalwork",8), hyponyms("work",2,"chef-d'oeuvre",7), hyponyms("work",2,"masterpiece",7), hyponyms("work",2,"leatherwork",6), hyponyms("work",2,"lacquerware",5), hyponyms("work",2,"lacework",4), hyponyms("work",2,"ironwork",3), hyponyms("work",2,"handwork",2), hyponyms("work",2,"handiwork",2), hyponyms("work",2,"handcraft",2), hyponyms("work",2,"handicraft",2), hyponyms("work",2,"followup",1), hyponyms("work",2,"follow-up",1), hyponyms("work",3,"paper_route",11), hyponyms("work",3,"public_service",10), hyponyms("work",3,"services",9), hyponyms("work",3,"teleworking",8), hyponyms("work",3,"telecommuting",8), hyponyms("work",3,"piecework",6), hyponyms("work",3,"work_load",5), hyponyms("work",3,"workload",5), hyponyms("work",3,"sailing",4), hyponyms("work",3,"navigation",4), hyponyms("work",3,"seafaring",4), hyponyms("work",3,"ministry",3), hyponyms("work",3,"booking",2), hyponyms("work",3,"engagement",2), hyponyms("work",3,"coaching_job",1), hyponyms("work",3,"coaching",1), hyponyms("work",3,"service",7), hyponyms("work",6,"shop",32), hyponyms("work",6,"workshop",32), hyponyms("work",6,"waterworks",31), hyponyms("work",6,"test_bed",30), hyponyms("work",6,"tannery",29), hyponyms("work",6,"studio",27), hyponyms("work",6,"studio",28), hyponyms("work",6,"shop_floor",26), hyponyms("work",6,"shipyard",25), hyponyms("work",6,"roundhouse",24), hyponyms("work",6,"rope_yard",23), hyponyms("work",6,"ropewalk",23), hyponyms("work",6,"proving_ground",22), hyponyms("work",6,"oyster_park",21), hyponyms("work",6,"oyster_bank",21), hyponyms("work",6,"oyster_bed",21), hyponyms("work",6,"lumberyard",20), hyponyms("work",6,"location",19), hyponyms("work",6,"laundry",18), hyponyms("work",6,"science_laboratory",17), hyponyms("work",6,"science_lab",17), hyponyms("work",6,"research_laboratory",17), hyponyms("work",6,"research_lab",17), hyponyms("work",6,"laboratory",17), hyponyms("work",6,"lab",17), hyponyms("work",6,"ironworks",15), hyponyms("work",6,"glassworks",14), hyponyms("work",6,"gasworks",13), hyponyms("work",6,"smithy",12), hyponyms("work",6,"forge",12), hyponyms("work",6,"fish_farm",11), hyponyms("work",6,"piscary",10), hyponyms("work",6,"fishery",10), hyponyms("work",6,"farm",9), hyponyms("work",6,"drill_site",7), hyponyms("work",6,"creamery",6), hyponyms("work",6,"pit",5), hyponyms("work",6,"colliery",5), hyponyms("work",6,"exchange",4), hyponyms("work",6,"exchange",8), hyponyms("work",6,"telephone_exchange",4), hyponyms("work",6,"central",4), hyponyms("work",6,"brokerage",3), hyponyms("work",6,"brokerage_house",3), hyponyms("work",6,"beehive",2), hyponyms("work",6,"bakehouse",1), hyponyms("work",6,"bakeshop",1), hyponyms("work",6,"bakery",1), hyponyms("work",6,"job",16), hyponyms("work",7,"writing",1), word("work")}
